.golangci.yml

```diff
@@ -148,7 +148,6 @@ linters-settings:
       - name: unexported-return
       - name: unreachable-code
       - name: unused-parameter
```

Comment:
Not sure why we're relisting the rules that are enabled by default.
IIUC, we only need to list a rule that we want to disable, but maybe I'm missing sth.

Reply:
ah maybe array merging wasn't/isn't supported?